workflow/controller/operator.go

```diff
@@ -111,6 +119,16 @@ type failedNodeStatus struct {
 	FinishedAt   metav1.Time `json:"finishedAt"`
 }
 
+// ExceededQuota checks if the error message indicates an exceeded quota in the namespace.
+func ExceededQuota(n *wfv1.NodeStatus) bool {
+	return strings.Contains(n.Message, exceededQuotaString)
```

Comment:
This is not a preferable way to diagnose this error as node strings could be arbitrary